### ISSUE TITLE
database download manually

### ISSUE THREAD

**Issue by BinhongLiu** [steineggerlab/foldseek#110]
## Expected Behavior
Could the pre-processed PDB and Alphafold/Swiss-Prot database be downloaded manually?

## Current Behavior
Could not download the database with script `foldseek databases PDB pdb tmp`. 
And the link https://foldseek.steineggerlab.workers.dev/pdb100.tar.gz could not be opened, either.

## Steps to Reproduce (for bugs)
`foldseek databases PDB pdb tmp`

## Foldssek Output (for bugs)

Create directory tmp
databases PDB pdb tmp 

MMseqs Version:                 5.53465f0
Tsv                             false
Force restart with latest tmp   false
Remove temporary files          false
Compressed                      0
Threads                         8
Verbosity                       3

04/15 18:24:45 [NOTICE] Downloading 1 item(s)
 *** Download Progress Summary as of Sat Apr 15 18:25:46 2023 ***                                                                             
==============================================================================================================================================
[#f34170 0B/0B CN:1 DL:0B]
FILE: tmp/5535929346258916704/pdb.tar.gz
----------------------------------------------------------------------------------------------------------------------------------------------

 *** Download Progress Summary as of Sat Apr 15 18:26:46 2023 ***                                                                             
==============================================================================================================================================
[#f34170 0B/0B CN:1 DL:0B]
FILE: tmp/5535929346258916704/pdb.tar.gz
----------------------------------------------------------------------------------------------------------------------------------------------

 *** Download Progress Summary as of Sat Apr 15 18:27:46 2023 ***                                                                             
==============================================================================================================================================
[#f34170 0B/0B CN:1 DL:0B]
FILE: tmp/5535929346258916704/pdb.tar.gz
----------------------------------------------------------------------------------------------------------------------------------------------

 *** Download Progress Summary as of Sat Apr 15 18:28:46 2023 ***                                                                             
==============================================================================================================================================
[#f34170 0B/0B CN:1 DL:0B]
FILE: tmp/5535929346258916704/pdb.tar.gz
----------------------------------------------------------------------------------------------------------------------------------------------

 *** Download Progress Summary as of Sat Apr 15 18:29:46 2023 ***                                                                             
==============================================================================================================================================
[#f34170 0B/0B CN:1 DL:0B]
FILE: tmp/5535929346258916704/pdb.tar.gz
----------------------------------------------------------------------------------------------------------------------------------------------

[#f34170 0B/0B CN:1 DL:0B]                                                                                                                    
04/15 18:29:51 [ERROR] CUID#7 - Download aborted. URI=https://foldseek.steineggerlab.workers.dev/pdb100.tar.gz
Exception: [AbstractCommand.cc:340] errorCode=2 Timeout.

04/15 18:29:51 [NOTICE] Download GID#f34170de50d28f9a not complete: tmp/5535929346258916704/pdb.tar.gz

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
f34170|ERR |       0B/s|tmp/5535929346258916704/pdb.tar.gz

Status Legend:
(ERR):error occurred.

aria2 will resume download if the transfer is restarted.
If there are any errors, then see the log file. See '-l' option in help/man page for details.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:02:08 --:--:--     0curl: (7) Failed to connect to foldseek.steineggerlab.workers.dev port 443: Connection timed out
--2023-04-15 18:32:00--  https://foldseek.steineggerlab.workers.dev/pdb100.tar.gz
Resolving foldseek.steineggerlab.workers.dev (foldseek.steineggerlab.workers.dev)... 31.13.91.6, 2a03:2880:f10d:83:face:b00c:0:25de
Connecting to foldseek.steineggerlab.workers.dev (foldseek.steineggerlab.workers.dev)|31.13.91.6|:443... failed: Connection timed out.
Connecting to foldseek.steineggerlab.workers.dev (foldseek.steineggerlab.workers.dev)|2a03:2880:f10d:83:face:b00c:0:25de|:443... failed: Network is unreachable.
Error: Could not download https://foldseek.steineggerlab.workers.dev/pdb100.tar.gz to tmp/5535929346258916704/pdb.tar.gz

## Context
A virtual environment was created and the foldseek was installed with conda `conda install -c conda-forge -c bioconda foldseek`.

## Your Environment
Linux

**Comment by milot-mirdita** [comment]
It seems to work correctly for me. Could you provide any details about your network connection? Do you have an alternative internet connection or VPN to try to download? I don't see a reason why the download should not work.

**Comment by BinhongLiu** [comment]
Thanks for your reply and kind help!
I also could not open the https://foldseek.steineggerlab.workers.dev/pdb100.tar.gz with the edge browser. I think it might because of the network barrier in my country. Do you have any alternate website to download the database, like ftp, cloud drive, or something other?

**Comment by milot-mirdita** [comment]
This seems to be a known issue, that I was not aware:
https://community.cloudflare.com/t/worker-is-not-available-in-china/497419
https://community.cloudflare.com/t/cloudflare-workers-suspected-of-being-blocked-in-china/382155

I guess we could try to set up a custom domain name. I can see how difficult that is maybe next week. Until then I don't really have a good solution for you.

**Comment by jiaweiguan** [comment]
> This seems to be a known issue, that I was not aware: https://community.cloudflare.com/t/worker-is-not-available-in-china/497419 https://community.cloudflare.com/t/cloudflare-workers-suspected-of-being-blocked-in-china/382155
> 
> I guess we could try to set up a custom domain name. I can see how difficult that is maybe next week. Until then I don't really have a good solution for you.

Hi, @milot-mirdita , I also encountered the same problem. Is there a new download address available now?

**Comment by milot-mirdita** [comment]
Could either of you @jiaweiguan or @BinhongLiu please try if the following URL works:
https://foldseek-cdn.mmseqs.org/afdb_swissprot.tar.gz

Please also check that the original URL *DOES NOT* work:
https://foldseek.steineggerlab.workers.dev/pdb100.tar.gz

**Comment by pengtb** [comment]
> Could either of you @jiaweiguan or @BinhongLiu please try if the following URL works: https://foldseek-cdn.mmseqs.org/pdb100.tar.gz
> 
> Please also check that the original URL _DOES NOT_ work: https://foldseek.steineggerlab.workers.dev/pdb100.tar.gz

Thanks. It works.

**Comment by milot-mirdita** [comment]
@pengtb, can you confirm that the second link DOES NOT work for you?

**Comment by pengtb** [comment]
> @pengtb, can you confirm that the second link DOES NOT work for you?
not really. it works just now. it seems that the original one is just unstable in CN somehow.

**Comment by milot-mirdita** [comment]
So the second link works sometimes and the first link works always?

**Comment by martin-steinegger** [comment]
@pengtb any update on this?

**Comment by martin-steinegger** [comment]
Consider this closed for now.
